tests/test_basics.py

```diff
@@ -1,8 +1,30 @@
-import pandas as pd
 import altair as alt
+import pandas as pd
+from fasthtml.common import to_xml
+
 from fh_altair import altair2fasthtml
 
+
 def test_no_err():
-    pltr = pd.DataFrame({'y': [1, 2, 3, 2], 'x': [3, 1, 2, 4]})
-    chart = alt.Chart(pltr).mark_line().encode(x='x', y='y').properties(width=400, height=200)
+    pltr = pd.DataFrame({"y": [1, 2, 3, 2], "x": [3, 1, 2, 4]})
+    chart = (
+        alt.Chart(pltr)
+        .mark_line()
+        .encode(x="x", y="y")
+        .properties(width=400, height=200)
+    )
     return altair2fasthtml(chart)
+
+
+def test_vega_options_passed_to_vegaEmbed():
```

Comment:
Bit of a nit, but maybe parametrize this test so we loop over all the options?

Reply:
Good advice! Thank you for the example. I think I misunderstood your original suggestion to somehow use parameterize to combine both `test_no_err` and `test_vega_options_passed_to_vegaEmbed` into a single test. 

Instead it's clear to me now that you were suggesting to use parameterize to go pass all combinations of inputs solely to `test_vega_options_passed_to_vegaEmbed`, that's straightforward and I can make that change.